.travis.yml

```diff
@@ -38,6 +52,20 @@ matrix:
       compiler: clang
       env: [CONFIGURATION=Release, *gcc-warnings]
 
+  # Sporadic failure builds
+    - compiler: gcc
+      env: CONFIGURATION=Debug    CTEST_OPTS="--repeat-until-fail 25 --schedule-random"
+      install: *install_cmake35
+      cache: *cache_homedir
+    - compiler: gcc
+      env: CONFIGURATION=Release  CTEST_OPTS="--repeat-until-fail 25 --schedule-random"
+      install: *install_cmake35
+      cache: *cache_homedir
+
+  allow_failures:
+    - env: CONFIGURATION=Debug    CTEST_OPTS="--repeat-until-fail 25 --schedule-random"
+    - env: CONFIGURATION=Release  CTEST_OPTS="--repeat-until-fail 25 --schedule-random"
+
```

Comment:
Oh, right. Those are named yaml anchors, and you could in fact define two anchors for the env keys.

Effectively, the result would be:

```yml
  # Sporadic failure builds
    - compiler: gcc
      env: &env_sporadic_dbg CONFIGURATION=Debug    CTEST_OPTS="--repeat-until-fail 25 --schedule-random"
      install: *install_cmake35
      cache: *cache_homedir
    - compiler: gcc
      env: &env_sporadic_rel CONFIGURATION=Release  CTEST_OPTS="--repeat-until-fail 25 --schedule-random"
      install: *install_cmake35
      cache: *cache_homedir

  allow_failures:
    - env: *env_sporadic_dbg
    - env: *env_sporadic_rel
```

Sounds good enough?